subsys/bluetooth/controller/Kconfig

```diff
@@ -201,19 +201,19 @@ choice BT_CTLR_TX_PWR
 
 config BT_CTLR_TX_PWR_PLUS_8
 	bool "+8 dBm"
-	depends on SOC_NRF52840
+	depends on SOC_NRF52820 || SOC_NRF52833 || SOC_NRF52840
```

Comment:
@MarekPieta I see now why this ended up as outdated. I think we should apply the same pattern as BT_CTLR_PHY_2M_SUPPORT and HAS_HW_NRF_RADIO_BLE_2M.
That way it is easier to manage the HW supported features.

Reply:
Changed

Reply:
Slightly different than what I meant. Now this is still nRF specific.

BT_CTLR_PHY_2M depends on BT_CTLR_PHY_2M_SUPPORT

select BT_CTLR_PHY_2M_SUPPORT if HAS_HW_NRF_RADIO_BLE_2M

Although maybe this could be left as  nRF specific additions?
Otherwise I think there should be a support label per TX power level, as it is now they are selected as a block.

Reply:
I preferred to leave it as nRF specific addition for simplicity. I can change it if you prefer adding a single support label per TX power.

Reply:
I'm fine with leaving it nRF specific for the moment.